src/components/VariantDb/fetchVariantStats.tsx

```diff
@@ -0,0 +1,10 @@
+import { apiInitialized } from '../../services/api';
+import { arrangerUpdateApiRoot } from 'common/injectGlobals';
+
+export const fetchVariantDBStats = async () =>
```

Comment:
why not in service folder? I prefer what you did, but it's not the convention in the app